### PR TITLE
Portfolio: Desktop version

### PR DESCRIPTION
**Pull request by DianaBeki**
- Update documentation.
- Linters config
- Use minimum width of 778px in style.css for desktop version.
- Replace < section > with < div > element in index.html
- Use flexbox layout to style cards items.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -76,21 +78,22 @@ <h2 class="heading-2">Tonic</h2>
...
+    <div class="card-1">
     <h2 class="heading-2">Multi-Post<br>Stories</h2>
-
     <div class="specify">
     <p class="card-text-1">Canopy</p>
     <img  class="dot" src="./images/dot.png" alt="dot img">
     <p class="card-text">Back End Dev</p>
     <img class="dot" src="./images/dot.png" alt="dot img">
     <p class="card-text">2015</p>
     </div>
```
Comment (Kamba56) [inline index.html]: - [x] Kindly fix the spacing between the different parts of your work cards to match the given template. You are expected to stick to design as much as possible.
### Your design
image

### Figma template
image

File: index.html [hunk 2]
```diff
@@ -218,9 +231,9 @@ <h1 class="heading1"  >Contact me</h1>
     <input type="text" value="Diana" name="name" maxlength="30"  placeholder="Name" class="input text" required/>
     <input type="email" value="dianab21@gmail.com" name="email" placeholder="Email" class="input text" required/>
     <textarea id="message" cols="30"  maxlength="500"   rows="9" placeholder="Write Your Message Here" required></textarea> 
-
+​
     <button type="submit" class="form-btn">Get In Touch</a></button>
   </form>
```
Comment (Kamba56) [inline index.html]: - [ ] Kindly fix the background image on the contact section to match the given template. I see you added the background image for the mobile on the desktop too. Consider downloading the desktop background and add it to your work.
### Your design
image
### Figma template
image

File: index.html [hunk 3]
```diff
@@ -13,19 +13,20 @@
...
-   <img src="./images/union-img.png" alt="union icon">
+   <img id="union" src="./images/union-img.png" alt="union icon">
   </nav> 
 </header> 
-  <section class="bio">
+<div class="header-section">
+  <div class="bio">
     <h1 class="heading">
       I’m Diana<br>
 Glad to see you!
```
Comment (Kamba56) [inline index.html]: - [x] Kindly center the contents of the headline section in the desktop view to match the given template. You are expected to stick to design as much as possible.
### Your design
image
### Figma template
image

File: style.css [hunk 4]
```diff
@@ -88,19 +85,27 @@ li {
...
+
+  /* transition: transform  4s cubic-bezier(.57,.64,1,.01); */
 }
 
+/* .heading:hover {
+  color: indigo;
+  transform: scaleX(1.1) translate(70px);
+  font-style: italic;
+  font-size: 35px;
+} */
```
Comment (Kamba56) [inline style.css]: - [ ] Please remove all commented code in your documents for a cleaner document.

File: style.css [hunk 5]
```diff
@@ -459,3 +463,148 @@ textarea {
...
+    gap: 3rem;
+    list-style: none;
+  }
+
+  .landing-page {
+    background-color: #fff;
+    border-bottom-left-radius: 10rem;
+    margin-bottom: 6rem;
+
+    /* height: 100vh; */
```
Comment (Kamba56) [inline style.css]: - [x] Kindly remove all commented codes from your documents. It is bad practice to leave commented code in your document.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @DianaBeki, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed
- [x] Great job implementing a media query
- [x] Great job adding desktop design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DianaBeki) [reply index.html]: Hello @Kamba56 Thanks for the feedback. I have fixed the spacing now. Can you kindly confirm if it's better?

File: index.html (hunk 3 above)
Comment (DianaBeki) [reply index.html]: Hello @Kamba56. Thanks for the feedback. I have fixed the spacing now. Can you kindly confirm if it's better?

File: style.css (hunk 5 above)
Comment (DianaBeki) [reply style.css]: Hello @Kamba56. Thanks for the feedback. I have fixed the spacing now. Can you kindly confirm if it's better?

Comment (Kamba56) [reply style.css]: Alright, kindly submit for a re-review.

File: style.css [hunk 6]
```diff
@@ -482,21 +480,28 @@ textarea {
...
   .header-section {
     display: flex;
     flex-direction: column;
+    justify-content: center;
+    align-items: center;
     background: url(./images/desktop-background.png);
     background-size: cover;
     background-repeat: no-repeat;
     background-position: 100% 100%;
   }
```
Comment (bobb-Rob) [inline style.css:494]: - [ ] [OPTIONAL] Please consider adding the correct margin or padding to the toolbar on the left and right side to match the Figma template. The red marker indicates that there should be about 155px space on both sides. 

Your work  
image

Figma Design
image

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @DianaBeki, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**